### PR TITLE
Created Studio release blog post

### PR DESCRIPTION
**Pull request by tapadipti**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/blog/2021-06-02-introducing-dvc-studio.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+makes DVC Studio special is this connection to the Git ecosystem.
+
+Most ML engineers already use Git for code versioning. `dvc init`, `dvc add`,
+`dvc push` - these simple Git-like DVC commands are all you need to convert your
+Git repos into DVC repos - a single source of truth for not just your code but
+also your data, model and metrics.
+
+Even this sounds like a lot? Well, you can start by simply visualizing metrics
+you add in custom json or csv files - DVC Studio works with these custom files
+as well.
```
Comment (jendefig) [inline content/blog/2021-06-02-introducing-dvc-studio.md:101]: Suggestion Reworked for flow from Header and changed last section.....  

Use Git for ML metrics tracking. Nothing fancy. Nothing difficult.

Most ML engineers already use Git for code versioning. `dvc init`, `dvc add`, `dvc push` - these simple Git-like DVC commands are all you need to convert your Git repos into DVC repos - a single source of truth for not just your code, but also your data, model and metrics. 

What makes DVC Studio special is this connection to the Git ecosystem.  The table and visuals in DVC Studio aren’t magic - they are simply a representation of the data in JSON or CSV files in your Git repositories. 

See how DVC Studio works with these file types in this video.

File: content/blog/2021-06-02-introducing-dvc-studio.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+## Automate your ML process. No-code.
+
+Mature ML teams reuse their code over and over again while tuning data and
+parameters. DVC-Studio automates this in the visual user interface. Running an
+experiment on Studio is as simple as it can get - simply modify the ML model
+hyperparameters and dataset version from a UI and save the changes with a
+comment. The modifications and the message will be automatically converted to a
+proper Git commit with the source code changes and the commit message. Your team
+members can see the changes through Git or GitHub/GitLab, and track the author
```
Comment (jendefig) [inline content/blog/2021-06-02-introducing-dvc-studio.md:122]: suggestion:  "through your Git platform"

**Review by jendefig — APPROVED** [review]
I left some comments to take into consideration for changing.  Otherwise it looks great!  And the image turned out 🔥.  Way to go @tapadipti !!!

File: content/blog/2021-06-02-introducing-dvc-studio.md (hunk 2 above)
Comment (tapadipti) [reply content/blog/2021-06-02-introducing-dvc-studio.md]: Changed. Thanks.

File: content/blog/2021-06-02-introducing-dvc-studio.md (hunk 1 above)
Comment (tapadipti) [reply content/blog/2021-06-02-introducing-dvc-studio.md]: Changed.

**Comment by tapadipti** [comment]
Thanks so much @jendefig for your review and comments :) . Have incorporated them.

**Comment by shcheklein** [comment]
@tapadipti Main image viewport width is 850px. We need to adjust main image size (and optimize it using something like tinypng) to be 1700px at least.

**Comment by tapadipti** [comment]
@shcheklein I've resized and optimized the images. Main image is 1700px and the experiments table image is 1600px (hope that works). Let me know if the resolution is too low.

**Review by shcheklein — APPROVED** [review]

**Pull request merged**